app/src/main/res/values-es/about.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">Acerca De</string>
+    <string name="about_close">Cerrar</string>
+    <string name="about_version">Soy monerujo %1$s (%2$d)</string>
+
+    <string name="donation_text">
+        \"¡Donar bastardos desagradecidos!\"
+    </string>
+
+    <string name="donation_address_label">Dirección de Donaciones</string>
```

Comment:
Dirección para donaciones *(literally address for donations)*

Reply:
Definitely this is a better translation. Thanks!